packages/devops/scripts/lambda/resources/startup.sh

```diff
@@ -19,9 +28,10 @@ echo "Starting up ${DEPLOYMENT_NAME} (${BRANCH})"
 mkdir -p $LOGS_DIR
 
 # Turn on cloudwatch agent for prod and dev (can be turned on manually if needed on feature instances)
-if [[ $DEPLOYMENT_NAME == "production" || $DEPLOYMENT_NAME == "dev" ]]; then
-    $DEPLOYMENT_SCRIPTS/startCloudwatchAgent.sh | while IFS= read -r line; do printf '\%s \%s\n' "$(date)" "$line"; done  >> $LOGS_DIR/deployment_log.txt
-fi
+# TODO currently broken
```

Comment:
This was already broken, but now that we're using pipefail on this script, it matters. We haven't really got a good solution for pushing telemetry to cloudwatch now that our instances are short-lived.